design/batch_retrieval_design.md

```diff
@@ -288,7 +288,31 @@ The following integration tests will be implemented in Secrets Provider for K8s
 
 ## Performance tests
 
---
+The following performance tests will be run using *Jmeter*.
```

Comment:
@abrahamko why are we falling back to Jmeter? Does the Conjur env already exist in Jmeter?

Reply:
We do not "fall back" to jmeter.
Jmeter is the tool to use to run the tests and analyze the results.
The conjur should still be deployed separately.

Reply:
When I say fall back I am saying that this is not how we did it for SP so I ask why we chose Jmeter. Suggesting to write a one-liner explaining this choice. For example "We chose to use Jmeter because the test environment is already setup and is the easiest to get end-to-end"